src/decls/types.js

```diff
@@ -93,3 +93,22 @@ declare type TChannelSpan = {
     parentid: Array<number>,
     flags: number
 };
+
+declare type ReporterConfig = {
+    flushIntervalMs: ?number,
```

Comment:
its seconds in go, and python, but  milliseconds in java.  I think this needs to be different depending on what the timers for each language take.